camelot/admin/action/list_action.py

```diff
@@ -1098,23 +1098,19 @@ def _get_state(self, model_context, filter_value):
         state.modes = modes = []
         if len(filter_value) is not None:
             state.notification = True
+        # TODO: only show clear filter mode if any filters are active
+        #if len(filter_value):
```

Comment:
Deze enkel in de lijst met modes tonen als er een filter actief is leek mij een logische verbetering, maar een verandering in de lijst met modes wordt niet opgepikt in de view. Is dat een gekend issue, of neem ik dat best met Tim op?

Reply:
idd best opnemen met Tim, mogelijk alternatief is dat die 'disabled' w, of zo

maar in principe zou een wijziging vh aantal modes wel moeten werken

(nb : Tim heeft wel een PR klaar staan waar heel die boel in aangepast w)

Reply:
Ter bevestiging: is opgelost in Tim's PR.